modules/restricted-guests/synapse/synapse_guest_module/__init__.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2023 Nordeck IT + Consulting GmbH
+# Copyright 2025 New Vector Ltd.
```

Comment:
Does this not need dual-licensing for the Element Commercial License? https://github.com/element-hq/element-modules/tree/main/packages/element-web-module-api#copyright--license

Reply:
Good point, I believe it does

Reply:
2890cc1